packages/paste-core/components/heading/__tests__/heading.test.tsx

```diff
@@ -1,107 +1,96 @@
 import * as React from 'react';
-import renderer from 'react-test-renderer';
-import {render} from 'react-dom';
+import {render, screen} from '@testing-library/react';
 import {Theme} from '@twilio-paste/theme';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {matchers} from 'jest-emotion';
 // @ts-ignore typescript doesn't like js imports
 import axe from '../../../../../.jest/axe-helper';
 import {Heading} from '../src';
 
+expect.extend(matchers);
+
 describe('Heading', () => {
-  it('should render an H1 at fontSize90', (): void => {
-    const tree = renderer
-      .create(
-        <Theme.Provider theme="console">
+  describe('Render', () => {
+    it('should render an H1 at fontSize90', (): void => {
+      const {asFragment} = render(
+        <Theme.Provider theme="default">
           <Heading as="h1" variant="heading10">
             This is an H1
           </Heading>
         </Theme.Provider>
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
+      );
+      expect(asFragment()).toMatchSnapshot();
```

Comment:
❓ Not blocking, but maybe we could phase out the snapshot tests as we touch these older test files?

Reply:
Yeah. I went back and forth on these. I feel like the snapshots make sense here, since we're verifying the element is rendering as a certain heading level, along with certain styles. I could be convinced otherwise though.

Reply:
It's be nice if we can. But not a blocker